.changes/next-release/feature-s3-78494.json

```diff
@@ -0,0 +1,5 @@
+{
+  "description": "All S3 requests will now use SigV4 by default so that we can take advantage of stronger security algorithms and improved access key isolation.",
+  "type": "feature",
```

Comment:
The question: do we want to minor version bump for this? There precedence for that sort of thing (changing to use s3transfer for instance).